### PR TITLE
OPENNLP-1399 - Integrate ASF Matomo into OpenNLP website

### PR DESCRIPTION
**Pull request by rzo1**
as the title says.

https://matomo.privacy.apache.org

### CODE REVIEW

File: src/main/jbake/templates/footer.ftl [hunk 1]
```diff
@@ -40,5 +40,24 @@
...
 	
 	</script>
+
+    <!-- Matomo -->
+    <script>
+        var _paq = window._paq = window._paq || [];
+        /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+        /* We explicitly disable cookie tracking to avoid privacy issues */
+        _paq.push(['disableCookies']);
+        /* Measure a visit to flink.apache.org and nightlies.apache.org/flink as the same visit */
```
Comment (kinow) [inline src/main/jbake/templates/footer.ftl]: I think the comment above was copy-pasted, but it's actually opennlp.apache.org (there's no opennlp under nightlies).

File: src/main/jbake/templates/footer.ftl [hunk 2]
```diff
@@ -40,5 +40,24 @@
...
+        _paq.push(['trackPageView']);
+        _paq.push(['enableLinkTracking']);
+        (function() {
+            var u="//analytics.apache.org/";
+            _paq.push(['setTrackerUrl', u+'matomo.php']);
+            _paq.push(['setSiteId', '33']);
```
Comment (kinow) [inline src/main/jbake/templates/footer.ftl:55]: https://analytics.apache.org/index.php?module=CoreHome&action=index&date=yesterday&period=day&idSite=33

:+1: thanks!

**Review by kinow — COMMENTED** [review]
Looking good! Thanks @rzo1 !

File: src/main/jbake/templates/footer.ftl (hunk 1 above)
Comment (rzo1) [reply src/main/jbake/templates/footer.ftl]: Yes, this is true. It is copy pasted from the Privacy Slack, there Martjin Visser posted the snippet for us. I will change the flink comment ;)

Comment (kinow) [reply src/main/jbake/templates/footer.ftl]: Thanks! I think this is the first ASF I participate that I know Matomo is used. Looking forward to learning how it works and maybe suggest to other projects I contribute to to try that too.

Comment (rzo1) [reply src/main/jbake/templates/footer.ftl]: Maven is using it for quite a while now and the list is growing but it isn't promoted across the TLPs.

**Review by jzonthemtn — APPROVED** [review]

**Review by kinow — APPROVED** [review]
Thanks!

**Comment by kinow** [comment]
Just waiting for Travis (:man_shrugging: even though it shouldn't matter for this PR) before merging (others can merge it now if they prefer).

**Comment by rzo1** [comment]
> Just waiting for Travis (man_shrugging even though it shouldn't matter for this PR) before merging (others can merge it now if they prefer).

Guess we need to migrate away from Travis CI any time soon. I read, that ASF discontinous to use it, so we obviously need to change that in the future.

**Comment by kinow** [comment]
True, LGTM. I think I saw an email from... INFRA? Telling projects to migrate away from Travis. Merging and updating JIRA. Thanks!

**Comment by jzonthemtn** [comment]
@rzo1 @kinow I could be wrong, but I don't think Travis CI is doing anything here and we can just delete the `.travis.yml` file. Pushing a commit triggers build bot and it does the deploy.

**Comment by kinow** [comment]
> @rzo1 @kinow I could be wrong, but I don't think Travis CI is doing anything here and we can just delete the `.travis.yml` file. Pushing a commit triggers build bot and it does the deploy.

Sounds good to me. Either delete it or replace by GH Actions to just build JBake/Maven. Either way would be better than having Travis as it's really slow these days :+1:

**Pull request merged**